fs/fs.go

```diff
@@ -181,6 +188,7 @@ func NewFilesystem(root string, cfg config.Config, opts ...Option) (_ snapshot.F
 		entryTimeout:        entryTimeout,
 		negativeTimeout:     negativeTimeout,
 		orasStore:           store,
+		bgFetcher:           bgFetcher,
```

Comment:
``bgFetcher`` is part of ``layer.Resolver``. Why do we need ``bgFetcher`` as part of ``fileSystem``? I don't see ``filesystem.bgFetcher`` is invoked anywhere

Reply:
Not yet, but maybe in the future (e.g., dynamically adjusting rate limit).